sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
+
+We're rendering `QuestionCrudController`, but in the context of our `DashboardController`. When we go to this page, in order to get the CRUD config, EasyAdmin first calls `configureCrud` on our dashboard. Then it calls `configureCrud` on our specific ride controllers. In this case, `QuestionCrudController` is super powerful. It means that we can override things on a dashboard level, like set some CRUD settings to apply to all sections or for one specific `crudController`.
+
+I'll prove it! Go back to `AbstractDashboardController`. In `configureCrud`, every area has four pages. I'll hold "cmd" and click to open this `Crud` thing. There's some constants on top here. Every CRUD section has an index page. That's this in 'edit' page, a 'new' page, and also a 'detail' page. Each page can then refer to a set of actions. These are links or buttons. For example, on the index page, right now, we have an action for editing and an action for deleting. We *could* add more actions or remove those actions.
+
+You can see how this is configured down in `configureActions`. This is for the dashboard, so this is the action configuration that applies to all of our sections. You can see that for the index page. It adds new "Edit" and "Delete" actions for the detail page. There's `EDIT`, `INDEX`, and `DELETE`. And if you're on the edit page, you have the actions `SAVE_AND_RETURN` and `SAVE_AND_CONTINUE`. One thing you'll notice if you look closely here is that we *do* have a detail page, but nobody actually links to it. You don't see an action calling `DETAIL` on any of these pages. So the page exists, but it's not really used out of the box. So let's add that. 
+
+We'll go to `DashboardController` and it doesn't matter where but I'll go down to the bottom. I'm going to go to "Code Generate" or "cmd + N" on a Mac, and go to "Override Methods..."" and override `configureCrud`. The first thing we want to do is call `paren::configureCrud`. So it can instantiate that object.
+
+Oh! but before we do, we actually want to go to "Override Methods..." and select `configureActions`. Now we want to call the parent method that can create the actions object and set up all of those default actions for us. To do that, add an action to our index page. We can use that CRUD class call on page index.
+
+```terminal
+->add(Crud::PAGE_INDEX)
+```
+
+ We're in, and we can actually use a constant for this as well.
+ 
+ ```terminal
+Action::DETAIL
+ ```
+
+So on the index page, I want to add the action and then the `DETAIL` action. The result of this, when we refresh the `QuestionCreditController`, is that we have this "Show" link that goes to the `DETAIL` action, and we can go to any section. 
```

Comment:
I left the "so" here because it felt like you were re-stating what we just did for clarity. Seemed appropriate.